src/nyx/python/new_bindings_py.cpp

```diff
@@ -173,22 +173,36 @@ py::tuple featurize_directory_imp (
     // We're good to extract features. Reset the feature results cache
     theResultsCache.clear();
 
-    auto arrow_output = !pandas_output;
+    theEnvironment.use_arrow = !pandas_output;
```

Comment:
My traditional question here: are we supporting other Apache formats - "feather" and "parquet"? If yes, then "theEnvironment.use_arrow" should be called "theEnvironment.use_apache" ...